src/index.js

```diff
@@ -0,0 +1,88 @@
+import { helpers } from '@eik/common';
+import { join } from 'path';
+import fetch from 'node-fetch';
+
+const isUrl = (value = '') => value.startsWith('http');
+
+const fetchImportMaps = async (urls = []) => {
+    try {
+        const maps = urls.map((map) => fetch(map).then((result) => {
+            if (result.status === 404) {
+                throw new Error('Import map could not be found on server');
+            } else if (result.status >= 400 && result.status < 500) {
+                throw new Error('Server rejected client request');
+            } else if (result.status >= 500) {
+                throw new Error('Server error');
+            }
+            return result.json();
+        }));
+        return await Promise.all(maps);
+    } catch (err) {
+        throw new Error(
+            `Unable to load import map file from server: ${err.message}`,
+        );
+    }
+};
```

Comment:
I have a feeling I've seen this import map fetching code in a a few different places now. Could this be a candidate to be moved out into the common package?

Reply:
Yes. Its time to generalize this one now. I can move it.